apps/web/__tests__/support/pageObjects/ProductListPageObject.ts

```diff
@@ -77,10 +77,7 @@ export class ProductListPageObject extends PageObject {
   }
 
   goToProduct() {
-    cy.intercept('/plentysystems/getProduct').as('getProduct');
-    this.products.first().click();
-    cy.wait('@getProduct');
-
-    return this
+    this.products.first().wait(2000).click().wait(2000);
```

Comment:
Because BE data response&initialization time varies, 2 out of 3 times tests are failing without a cool-off period, so the (non optimal) solution was to wait a bit before moving on.